components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorService.java

```diff
@@ -225,6 +236,11 @@ public AccessTokenInfo getTokens(String appName, String grantType,
             log.error(errorMsg, e, ExceptionCodes.ACCESS_TOKEN_GENERATION_FAILED);
             throw new APIManagementException(errorMsg, e, ExceptionCodes.ACCESS_TOKEN_GENERATION_FAILED);
         }
+
+        if (log.isDebugEnabled()) {
+            log.debug("Received access token for " + appName + " application.");
```

Comment:
Please follow this pattern for debug logs - https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/KubernetesGatewayImpl.java#L181

In this way, you don't need to check whether debug is enabled.